### DISCUSSION TITLE
Recurring fees

### DISCUSSION THREAD

**Discussion by GerhardMes** [frappe/books#465] [Ideas]
I would like to use it for my club. At the club we use Recurring fees for memberships. (different ones, depending on the status).
This would be - for me - a worthy feature

**Comment by liyakatalilal** [comment]
Recurring documents (could be for invoices, expenses of utilities) is must have feature for small business accounting. This is because invoice and expenses are identical and repetitive.

If not option to duplicate a document of one click could also be a good choice.

**Reply by 18alantom** [reply]
@lalnco recurring invoices is not available yet but duplication is available: [docs](https://docs.frappebooks.com/entries/entries.html#duplicating-an-entry)
